packages/terra-application-navigation/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "terra-application-navigation",
+  "main": "lib/ApplicationNavigation.js",
+  "version": "0.0.0",
+  "description": "The Terra Application Navigation is a responsive, themeable layout for building applications. ApplicationNavigation is comprised of selectable navigation and utility elements with selection callbacks.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-application-navigation",
+    "ApplicationNavigation",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "devDependencies": {
+    "terra-application": "^0.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": "^2.4.0",
+    "terra-application": "^0.0.0"
```

Comment:
We should get terra-application released and update this to `^1.0.0` after that package is released.